packages/app/src/debug/DebugCancelledAlert.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <Alert
+    :title="t('debugPage.manuallyCanceled')"
+    status="warning"
+    :icon="ErrorOutlineIcon"
+    class="flex flex-col mb-24px w-full"
+  >
+    <div class="flex items-center">
+      <div>{{ t('debugPage.specsSkipped', {n: totalSpecs, totalSkippedSpecs}) }}</div>
+      <div
+        v-if="canceledByEmail && canceledByFullName"
+        class="rounded-full font-semibold bg-orange-500 h-3px mx-10px w-3px"
+      />
+      <div
+        v-if="canceledByEmail && canceledByFullName"
+        class="flex items-center"
+      >
+        <UserAvatar
+          :email="canceledByEmail"
+          class="h-20px mr-7px w-20px"
+          data-cy="canceled-by-user-avatar"
+        />
+        <div>{{ canceledByFullName }}</div>
+      </div>
+      <div
+        v-if="canceledAt"
+        class="rounded-full font-semibold bg-orange-500 h-3px mx-10px w-3px"
+      />
+      <div v-if="canceledAt">
+        {{ dayjs(canceledAt).utc().format('MMM D, YYYY h:mm A UTC') }}
```

Comment:
Just checking, what is the format of `canceledAt`? Do we really want to show the time this was canceled in UTC? Shouldn't we show it in their locale?

Reply:
`canceledAt` is a date string like this `2019-01-25T02:00:00.000Z`. Not sure, I was just going off of the Figma design which specifies UTC at the end of the timestamp [here](https://www.figma.com/file/PAOozuCxtfetqP8fUVvg0y/branch/QkCp2F69X0qyLo7qnYQ67e/Cypress-App?node-id=21215%3A209340&t=ZV0S7Uun2UNc4K3g-0)

Reply:
UTC is for machines, no humans - I don't think we actually should show the UTC time. The idea is given UTC, we can show the correct date time to the user based on their location. The opposite isn't always true - there's ambiguity introduced, which is why we store in UTC, but display in locale time.

Let me write this out, just to make sure I'm understanding it correctly. I always seem to get confused around timezones. Let me know if any of this looks incorrect.

`2019-01-25T:02:00:00.000Z` would mean the time 2AM on the 25th of January, 2019, in **Greenwich**. The `Z` means UTC, no offset means +0. 

Running your code:

```js
dayjs.utc('2019-01-25T02:00:00.000Z').format('MMM D, YYYY h:mm A UTC')
"Jan 25, 2019 2:00 AM UTC" 
```

This would only be correct for users in **Greenwich**. Everyone else would need to realize it's in UTC, then do the conversion themselves (not a good UX). 

I think we want to localize this. We will need the [timezone](https://day.js.org/docs/en/timezone/timezone) plugin. It won't add any overhead, according to the docs:

> Day.js supports time zone via the [Internationalization API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl) in [supported environments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#Browser_compatibility).

Example: I am writing this at 9:00 AM on the 29th of Dec in GMT+10. The current time right now, in Greenwich, is `2022-12-28T23:00:00Z` (it's 10 hours behind 9AM - makes sense, I am in GMT+10).

I tested it out, the times look correct to me. You can see using `utc(...).tz('Greenwich')` returns the same time, makes sense, since it's +0.

![image](https://user-images.githubusercontent.com/19196536/209884721-5170891f-fa2c-4053-8543-26d03da97804.png)

Reply:
Yeah that makes sense - not sure why the decision was made to display it in UTC. I'll bring this up for discussion.

Reply:
Outside of developers, it's unlikely anyone else knows what UTC actually means or is used for -- it's probably just a oversight during design. You *never* want to show UTC in an interface.